.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+
+<!-- 
+This template will help you to have a meaningful PR, please follow it and do not leave it blanket. 
+-->
+
+# PR Description 
+
+## Related Issue 
+
+<!-- 
+Please use this format to link other issues with thier numbers: Close #123 
+https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword
+-->
+
+## Checklist 
+
+- [ ] Did you add a function?
+- [ ] Did you add the tests?
+- [ ] Did you follow the steps we provided?
```

Comment:
I think it would be good to link the steps you are referring to, as to be as clear as possible :smile:  Would it be https://unify.ai/docs/ivy/overview/contributing.html ?

Reply:
For the steps, I am assuming the PR is for frontend functions. Then I think we can improve this template and have other options to add.

Reply:
What's your input about that?

Reply:
I would just try to add some relevant links, if there are multiple maybe number them like [[0] ](url) [[1] ](url) . We could also leave it if you want it to be more general or you are referring to the application instructions. What do you think?

Reply:
I thought to keep it general for two reason: 
- have a minimal experience now and talk with the team about it. 
- we have backend, frontend, format style right now and having them all in this will make the PR description too long. 
- Long stuff would not be read lol